tests/Concerns/ImportableTest.php

```diff
@@ -85,4 +85,31 @@ public function array(array $array)
 
         $import->import('csv-with-html-tags.csv', 'local', Excel::CSV);
     }
+
+    /**
+     * @test
+     */
+    public function can_import_a_simple_xlsx_file_with_ignore_empty_setting()
+    {
+        config()->set('excel.imports.ignore_empty', true);
+
+        $import = new class implements ToArray {
+            use Importable;
+
+            /**
+             * @param array $array
+             */
+            public function array(array $array)
+            {
+                Assert::assertEquals([
+                    ['test', 'test'],
+                    ['test', 'test'],
+                ], $array);
+            }
+        };
+
+        $imported = $import->import('import.xlsx');
```

Comment:
Maybe it would be good to use a file that has some empty rows and testing it 2x. Once with ignore_empty = false and once = true, so we have some proof of the difference. Would you be able to provide such a file?